alchemist/alchemist-implementationbase/src/main/kotlin/it/unibo/alchemist/kotlin/AlchemistExtensions.kt

```diff
@@ -9,6 +9,7 @@ package it.unibo.alchemist.kotlin
 
 import it.unibo.alchemist.model.implementations.times.DoubleTime
 import it.unibo.alchemist.model.interfaces.Time
+import java.awt.Point
```

Comment:
Uhmm... these would be better off in the GUI project

Reply:
Done